src/application/services/property.service.ts

```diff
@@ -81,6 +86,30 @@ class PropertyService implements IPropertyService {
     }
     return owner;
   }
+
+  public async getMyProperties(
+    ownerEmail: string,
+  ): Promise<ResponseDtos.PropertiesDto> {
+    const owner = await this.getOwnerFromEmail(ownerEmail);
+    const properties = await this.propertyRepository.findByOwnerId(owner.id);
+    return { properties: properties.map(PropertyFactory.toDto) };
```

Comment:
Do `properties.map((property) => PropertyFactory.toDto(property))` instead of just `PropertyFactory.toDto` because that loses the `this` context and so if the function has any reference to `this` inside of it (ie it acceses an internal variable) it will fail.